libs/api/books/src/lib/reading-list.service.ts

```diff
@@ -28,4 +28,12 @@ export class ReadingListService {
       return list.filter(x => x.bookId !== id);
     });
   }
-}
+
+  async finishBook(id: string, finishedBook: ReadingListItem): Promise<void> {
```

Comment:
Since the method is just updating the `finished` and `finishedDate` properties, it would be better to not take in the entire `ReadingListItem`, maybe just `Pick<ReadingListItem, 'finished', 'finishedDate>`.